components/Player/NotesTable.tsx

```diff
@@ -35,20 +57,70 @@ const Note: React.FunctionComponent<{ note: Notes }> = ({
 
 type Props = React.PropsWithChildren<{
   userId: number | undefined;
-  playerNotes: Notes[] | undefined;
+  defaultNotes: Notes[];
 }>;
 
-const NotesTable: React.FC<Props> = ({ playerNotes }) => {
+const NotesTable: React.FC<Props> = ({ userId, defaultNotes }) => {
+  const [phrase, setPhrase] = useState<string>(" ");
+  const [notes, setNotes] = useState<Notes[]>(defaultNotes);
+
+  useEffect(() => {
+    const getNotes = async (): Promise<Notes[]> => {
+      const response = await fetch(
+        `/api/notes/readMany?id=${userId}&search=${phrase}`,
+        {
+          method: "GET",
+          headers: { "content-type": "application/json" },
+          redirect: "follow",
+        }
+      );
+      const data = (await response.json()) as ReadManyNotesDTO;
+      return data.notes;
+    };
+    async function fetchData(): Promise<void> {
+      setNotes(await getNotes());
+    }
+    fetchData();
+  }, [phrase, userId]);
+
+  const [categoryToggles, setCategoryToggles] = useState<
+    Record<string, Category>
+  >(
+    Object.keys(CATEGORIES).reduce<Record<string, Category>>(
+      (obj, categoryName) => {
+        return {
+          ...obj,
+          [categoryName]: {
+            color: CATEGORIES[categoryName as keyof typeof CATEGORIES],
+            enabled: true,
+          },
+        };
+      },
+      {}
+    )
+  );
+
+  const toggleCategory = (categoryName: string): void => {
+    setCategoryToggles({
+      ...categoryToggles,
+      [categoryName]: {
+        ...categoryToggles[categoryName],
+        enabled: !categoryToggles[categoryName].enabled,
+      },
+    });
+  };
+
   return (
     <div>
       <div className="flex flex-row mt-8">
-        <div className="text-gray-600 w-4/6">
+        <div className="text-gray-600 w-4/6 pr-12">
+          <Icon type="search" className="absolute ml-4 mt-3" />
```

Comment:
let's add padding for the right too so that the "x" for the search doesn't go all the way to the end, so `mx-4`